### DISCUSSION TITLE
What is the third parameter of set for custom cache?

### DISCUSSION THREAD

**Discussion by Lydanne** [mikro-orm/mikro-orm#2350] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/39021696/140469472-2066fff6-0a0f-4669-bc5c-f1c80c905d60.png)

**Comment by B4nan** [comment, marked as answer]
That is used in `FileCacheAdapter` for invalidation. It works only with metadata cache and you will get the file system path to the entity definition file there.

https://github.com/mikro-orm/mikro-orm/blob/master/packages/core/src/cache/FileCacheAdapter.ts#L26
https://github.com/mikro-orm/mikro-orm/blob/master/packages/core/src/cache/FileCacheAdapter.ts#L62
